### PR TITLE
Lavaland Golems have a larger naming pool

### PR DESCRIPTION
**Pull request by coiax**
Free Golems now have (on average) 174.1 different names to choose from, and their material is used as a Forename.

Say hello to Diamond Hematite, Gold Ilmenite, Plasma Cheridite, Iron Slate and Uranium Boulder!

Some minor easter eggs in the golem list have been included, but I've forgotten what most of them were, so only geologists will notice at this point.

:cl: coiax
rscadd: Free Golems on Lavaland have been naming their newborn siblings with a wider range of geology terms
/:cl:

### CODE REVIEW

**Comment by coiax** [comment]
Should consider pushing this naming system into the naming helpers and letting Xenogolems do it too.

**Comment by TheRealGLH** [comment]
Aren't xenogolems all made of the same material, though?

**Comment by coiax** [comment]
Yeah, so they'd all have the name Adamantine X. Which seems fine, then you'd still be able to identify them as a golem via name, but they still have a unique name rather than a number.

File: code/_globalvars/lists/names.dm [hunk 1]
```diff
@@ -11,9 +11,10 @@ var/list/lizard_names_male = file2list("config/names/lizard_male.txt")
 var/list/lizard_names_female = file2list("config/names/lizard_female.txt")
 var/list/clown_names = file2list("config/names/clown.txt")
 var/list/mime_names = file2list("config/names/mime.txt")
+var/list/golem_names = file2list("config/names/golem.txt")
```
Comment (ChangelingRain) [inline code/_globalvars/lists/names.dm]: I should note that this will conflict with your other PR.

Comment (coiax) [inline code/_globalvars/lists/names.dm]: Ah, but my other PR is PROBABLY only going to be for april fools. Unless people want to keep it for badminry. And if either one is accepted to master, it's not a very complicated merge.

**Comment by KorPhaeron** [comment]
I'd rather xenobio golems remain with the current naming scheme so it's easier to distinguish free golems/they remain faceless xenobio servants.

Cool list of names. Not sure how I feel about the forname/human stuff but I will probably merge this at 24 hours

**Pull request merged**